Wikipedia/Code/WMFAppViewController.m

```diff
@@ -632,7 +635,10 @@ - (void)performDatabaseHousekeepingWithCompletion:(void (^)(NSError *))completio
         DDLogError(@"Error on cleanup: %@", housekeepingError);
         housekeepingError = nil;
     }
-    
+
+    /// Housekeeping for the new talk page cache
+    [SharedContainerCacheHousekeeping deleteStaleCachedItemsIn: @"Talk Page Cache"];
```

Comment:
Go ahead and use your new SharedContainerCacheCommonFileNames.talkPageCache struct here.